BoxItScript.js

```diff
@@ -0,0 +1,41 @@
+const args = process.argv.slice(2);
+const words = args;
+function drawLine(num) {
+    let res = '━'.repeat(num);
+    return res;
+}
+function drawTopBorder(num) {
+    let res = '┎' + drawLine(num) + '┒';
```

Comment:
Declaring `result` and then returning `result` is good form. For simple functions it may be also acceptable to declare and return on a single line, eg.

`return res = '┎' + drawLine(num) + '┒';`